config/locales/devise.fr.yml

```diff
@@ -11,15 +11,15 @@ fr:
         Si votre adresse email existe dans notre base de données, vous allez recevoir
         un email avec les instructions de confirmation dans quelques minutes.
     failure:
-      already_authenticated: "Vous êtes déjà autentifé·e."
-      inactive: "Votre compte n'est pas encore activé."
+      already_authenticated: "Vous êtes déjà autentifé(e)."
+      inactive: "Votre compte n’est pas encore activé."
       invalid: "%{authentication_keys} ou mot de passe invalide."
       locked: "Votre compte est vérouillé."
       last_attempt: "Il vous reste un essai avant le verouillage de votre compte."
       not_found_in_database: "%{authentication_keys} ou mot de passe invalide."
-      timeout: "Votre session a expiré. Veuillez vous reconnecter pour continuer."
-      unauthenticated: "Vous devez être indentifié·e pour pouvoir continuer."
-      unconfirmed: "Vous devez valider votre adresse email pour pouvoir continuer."
+      timeout: "Votre session a expirée. Veuillez vous reconnecter pour continuer."
```

Comment:
It should be `Votre session a expiré` or `Votre session est expirée`, but not `Votre session a expirée`

Reply:
Right about language fix. About devise translation, we should definitely use the dedicated gem (devise-i18n).